lib/note-content-editor.tsx

```diff
@@ -1188,7 +1190,7 @@ const mapStateToProps: S.MapState<StateProps> = (state) => ({
   note: state.data.notes.get(state.ui.openedNote),
   notes: state.data.notes,
   searchQuery: state.ui.searchQuery,
-  selectedSearchMatchIndex: state.ui.selectedSearchMatchIndex,
+  selectedSearchMatchIndex: state.ui.selectedSearchMatchIndex ?? 0,
```

Comment:
Setting the default to 0 as when we were comparing indexes in componentDidUpdate the initial render was comparing null to 0 which was unexpected.